### PR TITLE
StringToArrayArgumentProcessRectorTest : add failing test around Traversable

### PR DESCRIPTION
**Pull request by gnutix**
I finally managed to get a failing test. :)

### CODE REVIEW

File: packages/CodingStyle/tests/Rector/Use_/RemoveUnusedAliasRector/Fixture/skip_vich_annotation.php.inc [hunk 1]
```diff
@@ -24,34 +24,3 @@ class Entity
...
-     * @var string
-     *
-     * @ORM\Column(name="slug", type="string", length=50, unique=true)
-     * @Assert\NotBlank()
-     * @Assert\Length(max="50")
-     */
-    private $slug;
-}
-
-?>
```
Comment (TomasVotruba) [inline packages/CodingStyle/tests/Rector/Use_/RemoveUnusedAliasRector/Fixture/skip_vich_annotation.php.inc]: Need to rebase on `master` to prevent this

Comment (TomasVotruba) [reply packages/CodingStyle/tests/Rector/Use_/RemoveUnusedAliasRector/Fixture/skip_vich_annotation.php.inc]: This is how: https://help.github.com/en/articles/syncing-a-fork

I have original repo as `upstream` and then shortun in bash console that does:

```
git fetch upstream && git merge upstream/master && cb $1 && git rebase master
```

Comment (gnutix) [reply packages/CodingStyle/tests/Rector/Use_/RemoveUnusedAliasRector/Fixture/skip_vich_annotation.php.inc]: The branch is up-to-date. I've decided to simplify the fixtures files for those failing tests I've added, as both parts of the file (separated by `-----`) where identical.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**